sdk/identity/azure-identity/src/main/java/com/azure/identity/AzureCliCredentialBuilder.java

```diff
@@ -36,4 +40,35 @@ public AzureCliCredentialBuilder tenantId(String tenantId) {
     public AzureCliCredential build() {
         return new AzureCliCredential(tenantId, identityClientOptions);
     }
+
+    /**
+     * For multi-tenant applications, specifies additional tenants for which the credential may acquire tokens.
```

Comment:
This verbage seems a bit off in the context of the `AzureCliCredential`. Here's what we have for .NET:

> Specifies tenants in addition to the specified <see cref="TenantId"/> for which the credential may acquire tokens. Add the wildcard value "*" to allow the credential to acquire tokens for any tenant the logged in account can access. If no value is specified for TenantId" this option will have no effect, and the credential will acquire tokens for any requested tenant.